### PR TITLE
tsdb docs first pass

### PR DESCRIPTION
**Pull request by owen-d**

### CODE REVIEW

**Review by slim-bean — APPROVED** [review]
LGTM

File: docs/sources/storage/_index.md [hunk 1]
```diff
@@ -50,9 +50,17 @@ You may use any substitutable services, such as those that implement the S3 API
...
 ### Single-Store
 
+Single-Store refers to the using object storage as the storage medium for both Loki's index as well as its data ("chunks"). There are two supported modes:
+
+#### Suggested (2.8+): tsdb
+
+The [TSDB index store]({{< relref "../operations/storage/tsdb" >}}) improves query performance, reduces TCO and has the same feature parity as "boltdb-shipper".
+
+#### Deprecated (2.0+): boltdb-shipper
+
```
Comment (JStickler) [inline docs/sources/storage/_index.md:60]: Available in Loki 2.0 through 2.7, BoldDB (also known as `boltdb-shipper`) has been deprecated and will be removed in a future release.  This single store configurations for Loki utilizes the chunk store for both chunks and the index, requiring just one store to run Loki.

File: docs/sources/storage/_index.md [hunk 2]
```diff
@@ -50,9 +50,17 @@ You may use any substitutable services, such as those that implement the S3 API
 
 ### Single-Store
 
+Single-Store refers to the using object storage as the storage medium for both Loki's index as well as its data ("chunks"). There are two supported modes:
+
+#### Suggested (2.8+): tsdb
```
[the remark this answers is not in the document]
Comment (JStickler) [reply docs/sources/storage/_index.md:55]: Unless we're going to capitalize TSDB?  Have we decided on a standard yet?

**Review by JStickler — COMMENTED** [review]
[Docs review]  Couple of suggestions for ya.

File: docs/sources/operations/storage/tsdb.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+"TSDB" is the new index in Loki, generally available starting in Loki v2.8. It is heavily inspired and takes much code from Prometheus's TSDB [sub-project](https://github.com/prometheus/prometheus/tree/main/tsdb). For a deeper explanation you can read Owen's [blog post](https://lokidex.com/posts/tsdb/). The short version is that this new index is more efficient, faster, and more scalable. It also resides in object storage like the [boltdb-shipper]({{< relref "./boltdb-shipper" >}}) index which preceded it.
+
+## Example Configuration
+
+To get started using TSDB, add the following configurations:
+
+```yaml
+schema_config:
+  configs:
+    # Old boltdb-shipper schema. Included in example for reference but does not need changing.
```
[the remark this answers is not in the document]
Comment (owen-d) [reply docs/sources/operations/storage/tsdb.md:18]: Yes, it's possible. This section was meant to show how to live-migrate from boltdb->tsdb, does that help clarify?

File: docs/sources/operations/storage/tsdb.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+weight: 1000
+---
+
+# TSDB
+
+"TSDB" is the new index in Loki, generally available starting in Loki v2.8. It is heavily inspired and takes much code from Prometheus's TSDB [sub-project](https://github.com/prometheus/prometheus/tree/main/tsdb). For a deeper explanation you can read Owen's [blog post](https://lokidex.com/posts/tsdb/). The short version is that this new index is more efficient, faster, and more scalable. It also resides in object storage like the [boltdb-shipper]({{< relref "./boltdb-shipper" >}}) index which preceded it.
+
+## Example Configuration
+
+To get started using TSDB, add the following configurations:
```
[the remark this answers is not in the document]
Comment (owen-d) [reply docs/sources/operations/storage/tsdb.md:13]: I'll add something for this. For context, Loki only has two configuration files:
* the main one for for configuring loki itself
* an optional one for tenant-specific configurations

File: docs/sources/operations/storage/tsdb.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (JStickler) [reply docs/sources/operations/storage/tsdb.md:18]: Yes, that makes sense.  As a newbie, seeing configuration for something that we've just said was deprecated was confusing.  Maybe add a sentence explaining that this is an example of live-migration?

**Comment by owen-d** [comment]
Going to merge this. @JStickler: happy to address any followups in another PR! Thanks for the review.

**Pull request merged**